test/karma.conf.js

```diff
@@ -1,196 +1,41 @@
-module.exports = function(config) {
-  // build out a name for browserstack
-  // {TRAVIS_BUILD_NUMBER} [{TRAVIS_PULL_REQUEST} {PR_BRANCH}] {TRAVIS_BRANCH}
-  var browserstackName = process.env.TRAVIS_BUILD_NUMBER;
-
-  if (process.env.TRAVIS_PULL_REQUEST !== 'false') {
-    browserstackName += ' ' + process.env.TRAVIS_PULL_REQUEST + ' ' + process.env.TRAVIS_PULL_REQUEST_BRANCH;
```

Comment:
the karma config package is missing the initial whitespace here, so, the build number and PR value would be smooshed together.

Reply:
https://github.com/videojs/videojs-generate-karma-config/pull/8